client/notifications/index.jsx

```diff
@@ -138,10 +138,8 @@ export class Notifications extends Component {
 
 		switch ( event.data.action ) {
 			case 'openPanel':
-				// checktoggle closes panel with no parameters
-				this.props.checkToggle();
-				// ... and toggles when the 2nd parameter is true
-				this.props.checkToggle( null, true );
+				// Ensure panel is opened.
+				this.props.checkToggle( null, true, true );
```

Comment:
Since we started throttling the function to prevent simultaneous toggle calls cancelling each other out, I needed to update this service worker message handler. It previously called the toggle function back to back with different params to first ensure it was closed and then toggle it open. Instead, I added a third param so we can just call it once and still ensure it will be open.